packages/core/src/test/codewhisperer/commands/startSecurityScan.test.ts

```diff
@@ -264,8 +263,6 @@ describe('startSecurityScan', function () {
         await scanPromise
         assert.ok(securityScanRenderSpy.notCalled)
         assert.ok(securityScanStoppedErrorSpy.calledOnce)
-        const warnings = testWindow.shownMessages.filter(m => m.severity === SeverityLevel.Warning)
-        assert.ok(warnings.map(m => m.message).includes('Security scan failed. Error: Security scan stopped by user.'))
```

Comment:
Instead of deleting you can move this to a test where the scope is PROJECT

Reply:
oh, I missed this. @ctlai95 please feel free to send a followup PR with this change.